dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -177,15 +177,14 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
     {
         THROW_HR_IF_MSG(E_ILLEGAL_METHOD_CALL, AppModel::Identity::IsPackagedProcess(), "Not applicable for packaged applications");
 
-        THROW_HR_IF(E_INVALIDARG, (displayName != winrt::hstring{}) xor (iconUri != nullptr));
+        THROW_HR_IF(E_INVALIDARG, (displayName == winrt::hstring{}) || (iconUri == nullptr));
```

Comment:
Do we want to return a message to the developer if this throws?

Reply:
Also, can the iconUri be not null but empty?

Reply:
Windows::Foundation::Uri doesn't allow L"" to be taken in as an input, it throws incorrect parameter error.